packages/compass-connections/src/modules/telemetry.spec.ts

```diff
@@ -237,6 +239,11 @@ describe('connection tracking', function () {
         has_kms_azure: false,
       };
 
+      if (!is_srv) {
```

Comment:
Shouldn't valid mongodb+srv urls result in is_public_cloud: true and public_cloud_name to be filled in to be one of the clouds atlas runs on? Right now those all seem to fail dns lookup in mongodb-cloud-info. Before this PR it would set is_public_cloud to false and public_cloud_name to ''. Now it leaves those undefined. Neither are ideal, I think.

Reply:
I think mongodb+srv:// URLs should result in the same telemetry being sent as their resolved mongodb:// equivalents, yes.

Reply:
OK I added code to resolve those URLs now.